mypyc/doc/native_operations.rst

```diff
@@ -54,3 +54,5 @@ These variants of statements have custom implementations:
 * ``for ... in seq:`` (for loop over a sequence)
 * ``for ... in enumerate(...):``
 * ``for ... in zip(...):``
+* ``for ... in filter(...):``
```

Comment:
Have you considered supporting `list(filter(...))` as well -- this seems quite common (in a follow-up PR)?

Reply:
Fwiw this was part of the intent behind the list-built-from-range tests

I wasn't actually testing that we can build a list from a range, I was preparing IR to reflect how this helper would change the C implementation